pkg/network/ebpf/c/prebuilt/tracer.c

```diff
@@ -45,10 +52,10 @@ struct bpf_map_def SEC("maps/tcp_stats") tcp_stats = {
     .namespace = "",
 };
 
-/* Will hold the tcp close events
+/* Will hold the tcp/udp close events
  * The keys are the cpu number and the values a perf file descriptor for a perf event
  */
-struct bpf_map_def SEC("maps/tcp_close_event") tcp_close_event = {
+struct bpf_map_def SEC("maps/conn_close_event") conn_close_event = {
```

Comment:
I'd watch very closely the metric that tracks the number of perf events lost as you'll be writing more events to this. Since we consume it using a single goroutine, this might become a bottleneck.